avm/ptn/authorization/policy-assignment/main.bicep

```diff
@@ -111,7 +111,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
 }
 
 module policyAssignment_mg 'modules/management-group.bicep' = if (empty(subscriptionId) && empty(resourceGroupName)) {
-  name: '${uniqueString(deployment().name, location)}-PolicyAssignment-MG-Module'
```

Comment:
Just a remark @jtracey93 that the uniqueString() function always return a string of length 13. As such the use of deployment().name has no impact here as it's wrapped in the uniqueString(). That's good in that you can put whatever in that function, but also means that everything outside does have a direct impact. Hence it's good to shorten e.g. the term 'PolicyAssignment' to 'PolAssgn'. Just some input for trivia night 😉 feel free to resolve

Reply:
Thanks. I have shortened it in this PR. 

And the deployment name removal
Is due to the issue I have discovered with stacks as linked in the PR body. 

Can chat next week on it!